language-support/java/codegen/src/it/java-1.6/com/daml/lf_1_6/ContractKeysTest.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package com.daml.lf_1_5;
```

Comment:
Can’t this be part of the generic tests we run for all versions, same question for Optional, List, ContractKeys, ParametrizedContractId, and maybe Map (haven’t checked if that’s TextMap or not)? Then we can reduce probably 90% of the duplication ere.